ui/raidboss/data/04-sb/ultimate/ultima_weapon_ultimate.js

```diff
@@ -267,6 +267,28 @@ export default {
         },
       },
     },
+    {
+      id: 'UWU 爆岩',
+      netRegex: NetRegexes.startsUsing({ id: '2B6A', source: '爆破岩石', capture: false }),
```

Comment:
```suggestion
      id: 'UWU Titan Gaol Burst',
      netRegex: NetRegexes.startsUsing({ id: '2B6A', source: 'Bomb Boulder', capture: false }),
      netRegexCn: NetRegexes.startsUsing({ id: '2B6A', source: '爆破岩石', capture: false }),
```

The English name for this ability is Burst.